modules/rostests/apitests/apphelp/register.cpp

```diff
@@ -0,0 +1,374 @@
+/*
+ * PROJECT:     apphelp_apitest
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Tests for shim database registration
+ * COPYRIGHT:   Copyright 2017-2019 Mark Jansen (mark.jansen@reactos.org)
+ */
+
+#include <ntstatus.h>
+#define WIN32_NO_STATUS
+#include <windows.h>
+#include <ntndk.h>
+#include <atlbase.h>
```

Comment:
Why is this happening?
> fatal error: atlbase.h: No such file or directory

[Travis: gcc](https://travis-ci.org/reactos/reactos/builds/493461126#L723), [AppVeyor: msvc](https://ci.appveyor.com/project/AmineKhaldi/reactos/builds/22382518/job/r47t3nse3sj5hp1b?fullLog=true#L4729), [AppVeyor: msvc-x64](https://ci.appveyor.com/project/AmineKhaldi/reactos/builds/22382518/job/i21xonft72uuvsl3?fullLog=true#L5553), [AppVeyor: clang-cl](https://ci.appveyor.com/project/AmineKhaldi/reactos/builds/22382518/job/likw1j94qchcbnxd?fullLog=true#L6534)

Reply:
Either I forgot a dependency rule in CMake, or one of my other local changes that is not in this commit fixes it.

Reply:
Looks like all the other apitests that `#include <atlbase.h>` also have `${REACTOS_SOURCE_DIR}/sdk/lib/atl` in their `CMakeLists.txt` file's `include_directories(...)`, is that what needs to be done to fix this?